### PR TITLE
Added a new package service_os that can make Consul run as a service on Windows. 

### PR DESCRIPTION
**Pull request by MagnumOpus21**
This PR enables Consul to be run as a service on Windows, without additional changes to the core codebase, apart from import the package as a side-effect.

### CODE REVIEW

File: website/source/docs/guides/windows-guide.html.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+layout: "docs"
+page_title: "Windows Service"
+sidebar_current: "docs-guides-windows-service"
+description: |-
+  For our friends running Consul on Windows, we have good news. By using the _sc_ command either on Powershell or
```
Comment (mkeeler) [inline website/source/docs/guides/windows-guide.html.md]: I would probably remove the "For our friends running Consul on Windows, we have good news part" Both from here and down in the actual body of the doc.

File: website/source/docs/guides/windows-guide.html.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+The steps presented here assume, that the user has launched **Powershell** with _Adminstrator_ capabilities.
+
+If you come across bugs while using Consul for Windows, do not hesitate to open an issue [here](https://github.com/hashicorp/consul/issues).
+
+## Detailed steps involved in making Consul run as a service on Windows
+Download the Consul binary for your architecture.
+
+Setup your environmental _path_ variable, so that Windows can find 
+the Consul binary. (Will be handy for quick Consul commands)
```
Comment (mkeeler) [inline website/source/docs/guides/windows-guide.html.md]: "handy" -> "useful" 

To me useful sounds like a word more likely to be common for our international audience. I could be wrong though.

File: website/source/docs/guides/windows-guide.html.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                                    (STOPPABLE, NOT_PAUSABLE, ACCEPTS_SHUTDOWN)
+            WIN32_EXIT_CODE    : 0  (0x0)
+            SERVICE_EXIT_CODE  : 0  (0x0)
+            CHECKPOINT         : 0x0
+            WAIT_HINT          : 0x0
+            PID                : 8008
+            FLAGS              : 
+     ```
+If you followed the steps above, congratulations, you have successful made Consul 
+run as a service on Windows. The service automatically starts up during/after boot, so you don't need to
```
Comment (mkeeler) [inline website/source/docs/guides/windows-guide.html.md]: The whole "If you followed the steps above, congratulations, you have successful made Consul run as a service on Windows." could be removed.

Comment (MagnumOpus21) [reply website/source/docs/guides/windows-guide.html.md]: 👍

**Pull request merged**